incubator/vault/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.5.0
-appVersion: 0.9.0
+version: 0.5.1
```

Comment:
should this change trigger a `0.6.0` version?

Reply:
I considered it a small change so i just updated the minor version.  We can update it to `0.6.0` since I also updated the vault image being used.